src/com/owncloud/android/utils/DisplayUtils.java

```diff
@@ -207,8 +207,17 @@ else if ((System.currentTimeMillis() - time) < 60 * 1000) {
                 dateString = DateUtils.getRelativeDateTimeString(c, time, minResolution, transitionResolution, flags);
             }
         }
-        
-        return dateString.toString().split(",")[0];
+
+        String[] parts = dateString.toString().split(",");
+        if (parts.length == 2) {
+            if (parts[1].contains(":") && !parts[0].contains(":")) {
+                return parts[0];
+            } else if (parts[0].contains(":") && !parts[1].contains(":")) {
+                return parts[1];
+            }
+        }
+        //dateString contains unexpected format. use localized, absolute date.
+        return DisplayUtils.unixTimeToHumanReadable(time);

```

Comment:
`getRelativeDateTimeString` outputs something like `Dec 12, 4:12 AM` (except that date and time might be swapped, there is no comma, time might not contain colon, etc.) and if I interpret the documentation correctly it even shows the year for older dates. But there should be no seconds.
Actually, this function is pretty much what we want except that it shows additionally the time. So I think it is okay to use the whole output, if we are unable to determine what and where the time part is.

The seconds were a result from `unixTimeToHumanReadable` which outputs dates like ``Dec 31, 1969 4:00:00 PM``.